src/klein/test/test_exports.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tests for exported API.
+"""
+
+from twisted.trial import unittest
+
+
+
+class PublicSymbolsTestCase(unittest.TestCase):
+    """
+    Tests for public API modules.
+    """
+
+    def test_klein(self):
+        # type: () -> None
+        """
+        Test exports from L{klein}.
+        """
+        import klein as k
+        import klein._app as a
+        import klein._plating as p
+
+        self.assertIdentical(k.Klein, a.Klein)
+        self.assertIdentical(k.handle_errors, a.handle_errors)
+        # Blocked on https://github.com/twisted/klein/issues/224
+        # self.assertIdentical(k.resource, a.resource)
```

Comment:
I would prefer not to leave commented out code here.  I think it'd be better to link to this test in a diff from #224.